ci-operator/step-registry/openshift-power-monitoring/tests/kepler/openshift-power-monitoring-tests-kepler-commands.sh

```diff
@@ -44,22 +46,23 @@ main() {
 	echo "Running e2e tests"
 
 	log_events "$OPERATORS_NS" &
-	log_events "openshift-kepler-operator" &
+	log_events "$KEPLER_DEPLOYMENT_NS" &
 
 	local ret=0
 
-	./e2e.test -test.v -test.failfast 2>&1 | tee "$LOGS_DIR/e2e.log" || ret=1
+	./e2e.test -test.v -test.failfast -openshift 2>&1 | tee "$LOGS_DIR/e2e.log" || ret=1
```

Comment:
```suggestion
	./e2e.test -test.v -test.failfast 2>&1 | tee "$LOGS_DIR/e2e.log" || ret=1
```
Openshift should be default. The flag is for vanilla k8s to run tests.